inst/Upgrades.qml

```diff
@@ -0,0 +1,289 @@
+		ChangeRename{	from: "confidenceInterval";				to: "descriptiveTableCi"		}
+		ChangeRename{	from: "confidenceIntervalInterval";		to: "descriptiveTableCiLevel"	}
+		ChangeRename{	from: "countProp";						to: "descriptivesAs"			}
+		ChangeJS
+		{
+			name: "descriptivesAs"
+			jsFunction:	function(options)
+			{
+				switch(options["descriptivesAs"])
+				{
+					case "descCounts":		return "counts";
+					case "descProps":		return "proportions";
+				}
+			}
+		}
+		ChangeRename{	from: "descriptivesPlot";						to: "descriptivePlot"			}
+		ChangeRename{	from: "descriptivesPlotConfidenceInterval";		to: "descriptivePlotCiLevel"	}
+	}
+
+	Upgrade
+	{
+		functionName:		"MultinomialTestBayesian"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
+
+		ChangeRename{	from: "counts";					to: "count"				}
+		ChangeRename{	from: "exProbVar";				to: "expectedCount"		}
+		ChangeRename{	from: "hypothesis";				to: "testValues"		}
+		ChangeJS
+		{
+			name:		"testValues"
+			jsFunction:	function(options)
+			{
+				switch(options["testValues"])
+				{
+					case "multinomialTest":		return "equal";
+					case "expectedProbs":		return "custom";
+				}
+			}
+		}
+		ChangeRename{	from: "tableWidget";					to: "testValuesCustom"			}
+		ChangeRename{	from: "descriptives";					to: "descriptiveTable"			}
+		ChangeRename{	from: "credibleInterval";				to: "descriptiveTableCi"		}
+		ChangeRename{	from: "credibleIntervalInterval";		to: "descriptiveTableCiLevel"	}
+		ChangeRename{	from: "countProp";						to: "descriptivesAs"			}
+		ChangeJS
+		{
+			name: "descriptivesAs"
+			jsFunction:	function(options)
+			{
+				switch(options["descriptivesAs"])
+				{
+					case "descCounts":		return "counts";
+					case "descProps":		return "proportions";
+				}
+			}
+		}
+		ChangeRename{	from: "descriptivesPlot";						to: "descriptivePlot"			}
+		ChangeRename{	from: "descriptivesPlotCredibleInterval";		to: "descriptivePlotCiLevel"	}
+	}
+
+	Upgrade
+	{
+		functionName:		"ContingencyTables"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
+
+		ChangeRename{	from: "VovkSellkeMPR";							to: "vovkSellke"				}
+		ChangeRename{	from: "LogOdds";								to: "oddsRatioAsLogOdds"		}
+		ChangeRename{	from: "oddsRatioConfidenceIntervalInterval";	to: "oddsRatioCiLevel"			}
+		ChangeRename{	from: "oddsRatioHypothesis";					to: "oddsRatioAlternative"		}
+		ChangeSetValue
+		{
+			name:		"oddsRatioAlternative"
+			condition:	function(options) { return options["oddsRatioAlternative"] === "two.sided" }
+			jsonValue:	"twoSided"
+		}
+		ChangeRename{	from: "cochransAndMantel";						to: "cochranAndMantel"						}
+		ChangeRename{	from: "testOddsRatioEquals";					to: "cochranAndMantelCommonOddsRatioTest"	}
+		ChangeRename{	from: "hideSmallCounts";						to: "countsHiddenSmallCounts"				}
+		ChangeRename{	from: "hideSmallCountsLessThan";				to: "countsHiddenSmallCountsThreshold"		}
+		ChangeRename{	from: "zTestCompareColumns";					to: "zTestColumnComparison"					}
```

Comment:
What's zTestcolumnComparison?

Reply:
to be honest I have no idea, the component is specified like this
```qml
Group
{
	title: qsTr("Z-Test")
	debug: true
	CheckBox
	{
		name: "zTestColumnComparison"; label: qsTr("Compare column proportions")
		CheckBox { name: "zTestAdjustedPValues";	label: qsTr("Adjust p-values") }
	}
}
```

where I just changed the names according to our principles.

However as you can see the options are specified as `debug`, so they do not show in JASP, and there is no functionality associated with this in the R code. It's just a placeholder for something someone at some point planned to implement but it has not been done.

I could as well remove this entirely, but I did not want to potentially destroy someone else's plans with the functionality

Reply:
Let's just consider this resolved.